### PR TITLE
osquery 1.4.1

### PR DESCRIPTION
**Pull request by marpaia**
Updating the version of osquery in homebrew to 1.4.1

### CODE REVIEW

**Comment by marpaia** [comment]
It seems like the thrift command isn't available. `osquery.rb` depends_on thrift. Is there any reason why that wouldn't link in the "thrift" binary?

**Comment by marpaia** [comment]
oh, wait, we figured it out, update incoming.

File: Library/Formula/osquery.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 require "formula"
+require "fileutils"
```
Comment (jacknagel) [inline Library/Formula/osquery.rb]: not needed, formula already includes fileutils.

File: Library/Formula/osquery.rb [hunk 2]
```diff
@@ -44,6 +44,7 @@ def install
                                  "--record=installed.txt"}
     end
 
+    mkdir_p "#{buildpath}/generated"
```
Comment (jacknagel) [inline Library/Formula/osquery.rb]: this can just be `mkdir "generated"`

Comment (mikemcquaid) [inline Library/Formula/osquery.rb]: Also: why is this needed? Seems like something the CMake buildsystem should handle.

Comment (marpaia) [inline Library/Formula/osquery.rb]: we have some code generation that happens in a Makefile which wraps CMake.

**Comment by marpaia** [comment]
sorry for the delay @jacknagel, I addressed your comments and squashed into one commit.

File: Library/Formula/osquery.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Formula/osquery.rb]: Sure but you could have CMake make the generated directory as the first thing it does, no?

Comment (marpaia) [inline Library/Formula/osquery.rb]: We would have to cut a new release to add it and we're not ready to do that. For now, I've added an issue to track this and we'll be sure to update for 1.4.2: https://github.com/facebook/osquery/issues/773

**Comment by marpaia** [comment]
AFAICT, this is ready to merge. Let me know if you still need a response on anything.

File: Library/Formula/osquery.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Formula/osquery.rb]: Great, that's all I ask. Thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by marpaia** [comment]
thanks @mikemcquaid, always a pleasure.
